tools/releaseBuild/generatePackgeSigning.ps1

```diff
@@ -42,11 +42,11 @@ function New-FileElement
     if(Test-Path -Path $file)
     {
         $name = Split-Path -Leaf -Path $File
-        $null = $fileElement = $XmlDoc.CreateElement("file")
+        $fileElement = $XmlDoc.CreateElement("file")
```

Comment:
Please exclude this change from this PR.

Reply:
No, sorry this is a change that is needed.  Saw it in reverse when I talked to you.